installer/bundle_builder/config/ubuntu/22.04/k8s/1_22/etc/modules-load.d/containerd.conf

```diff
@@ -0,0 +1,2 @@
+overlay
```

Comment:
Why is this directory called `k8s/1_22`?

Reply:
Yea, I realised we don't need this directory..these configs are same as ubuntu 20